<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewUpload</class>
 <widget class="QDialog" name="DialogNewUpload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Upload</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="jobLabel">
       <property name="text">
        <string>&amp;Job Name</string>
       </property>
       <property name="buddy">
        <cstring>jobLine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="jobLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fileLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&amp;File</string>
          </property>
          <property name="buddy">
           <cstring>fileLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileLine"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="fileOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Open</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="tempDirLabel">
          <property name="text">
           <string>&amp;Temp. Dir.</string>
          </property>
          <property name="buddy">
           <cstring>tempDirLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tempDirLine"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="tempDirOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Open</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Security</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="encryptCheck">
        <property name="text">
         <string>&amp;encrypt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>&amp;Password</string>
          </property>
          <property name="buddy">
           <cstring>passwordLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLine">
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showPWcheck">
          <property name="text">
           <string>&amp;show</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="collectionCheck">
        <property name="text">
         <string>&amp;use Collection</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="imageDirLabel">
          <property name="text">
           <string>&amp;Image Folder</string>
          </property>
          <property name="buddy">
           <cstring>imageDirLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imageDirLine"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="imageDirOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Open</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>140</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>collectionCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageDirLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collectionCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageDirLine</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encryptCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encryptCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLine</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encryptCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>showPWcheck</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collectionCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageDirOpen</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
